### DISCUSSION TITLE
Is it possible to change the layout of the editor.

### DISCUSSION THREAD

**Discussion by TrevorBlythe** [GrapesJS/grapesjs#5219] [Q&A]
Ive spent the past week just trying to change the layout on "demo.html" but it seems impossible. The HTML of the layout just spawns out of nowhere and I can't find what makes it happen or how to change it. 

If anything I really need to know how to make it so that when you press the style manager button it doesn't disable the other panels

**Comment by artf** [comment]
Hi @TrevorBlythe the default UI of the editor is managed by the internal [Panels module](https://grapesjs.com/docs/api/panels.html). I'd suggest reading the [Getting Started](https://grapesjs.com/docs/getting-started.html) guide to have a quick overview on how to deal with the default UI

**Comment by sfreed** [comment]
I was able to change the layout of the editor. Basically, you create an html page with a bunch of divs on it and connect each of the different managers to the different divs. That allows you to change the layoug of where things render. I used a 3rd party set of libraries to manage the page (tabs, accorrdians, models, etc) with the provided API for many of the managers provided by Grapes.
![screen](https://github.com/GrapesJS/grapesjs/assets/13748756/178e1ea7-9e1f-43d8-ad01-830702f864ec)
